src/main/groovy/org/mockito/release/notes/GitNotesBuilder.java

```diff
@@ -41,10 +45,13 @@ public String buildNotes(String version, String fromRevision, String toRevision,
         ContributionsProvider contributionsProvider = Vcs.getContributionsProvider(Exec.getProcessRunner(workDir));
         ContributionSet contributions = contributionsProvider.getContributionsBetween(fromRevision, toRevision);
 
+        GitHubContributorsProvider contibutorsProvider = Contributors.getGitHubContibutorsProvider(authTokenEnvVar);
```

Comment:
I really like this separation of concerns, implementing the new functionality in separate set of interfaces / implementations. This makes the code easy to grasp and work with! I cannot wait to bust the release tools into a set of small libraries.